webpack/move_to_foreman/common/helpers.js

```diff
@@ -3,19 +3,31 @@ import { addToast } from 'foremanReact/redux/actions/toasts';
 const urlBuilder = (controller, action, id = undefined) =>
   `/${controller}/${id ? `${id}/` : ''}${action}`;
 
-const urlWithSearch = (base, searchQuery) =>
-  `/${base}?search=${searchQuery}`;
+const urlWithSearch = (base, searchQuery) => `/${base}?search=${searchQuery}`;
 
 const stringIsInteger = (value) => {
   // checking for positive integers only
   const reg = new RegExp('^[0-9]+$');
   return reg.test(value);
 };
 
+const getErrorMessegeByResponseMessage = (message) => {
+  const errorMessageHash = {
+    '404 Not Found': 'The subscription cannot be found upstream',
+    '410 Gone': 'The subscription is no longer available',
```

Comment:
Regardless of where this is going to be placed, the messages need to be translated.